scripts/run_e2e_tests.sh

```diff
@@ -173,7 +173,7 @@ node_modules/.bin/webdriver-manager start --versions.chrome 2.41 --detach --quie
 # TODO(jacob): Find a webdriver or selenium argument that controls log level.
 (node_modules/.bin/webdriver-manager start 2>/dev/null)&
 # Start a demo server.
-(python ../oppia_tools/google_appengine_1.9.67/google_appengine/dev_appserver.py --host=0.0.0.0 --port=9001 --clear_datastore=yes --dev_appserver_log_level=critical --log_level=critical --skip_sdk_update_check=true $APP_YAML_FILEPATH)&
+(python ../oppia_tools/google_appengine_1.9.67/google_appengine/dev_appserver.py --host=0.0.0.0 --port=9001 --clear_datastore=yes --dev_appserver_log_level=critical --log_level=critical --skip_sdk_update_check=true $APP_YAML_FILEPATH > log.txt)&
```

Comment:
We should be careful about creating files on developers' computers. Where is this one located? Could we put it in oppia_tools instead at a named path?

Reply:
Oops, my bad. This was introduced for debugging. It can be reverted back to as it was before.

Reply:
Removed.